src/govuk/settings/_typography-font.scss

```diff
@@ -52,6 +52,8 @@ $govuk-font-family-tabular: if(
 
 /// Font families to use for print media
 ///
+/// This option avoids font-related issues that occur with some printer drivers and operating systems.
```

Comment:
```suggestion
/// We recommend that you use system fonts when you print. This will avoid issues with some printer drivers and operating systems.
```